### DISCUSSION TITLE
Auth table missing from Triggers?

### DISCUSSION THREAD

**Discussion by CR1AT0RS** [supabase/supabase#2894] [Questions] [answered]
This sounds silly but based on my reading of other discussions, I am thinking there is a way to add a trigger to signup function. And I can do signup something like this:

- In the data object I can add full_name, addresses etc. during the initial signup request.
- - Is this not possible to send data object in initial request? and add full name from the signup request itself?
- -- I would like to send phone number, address etc in this object.

Signnup.jsx
```
 const { error, user } = await signUp({ email, password, **data**});
```

- After that I can create a function like this:

begin
  insert into public.users (id, full_name, avatar_url)
  values (new.id, new.raw_user_meta_data->>'full_name', new.raw_user_meta_data->>'avatar_url');
  return new;
end;
 

- Then I am trying to add a trigger whenever a row changes in auth table.

When I go to triggers to add this trigger and bind it to the auth table on any new insert I don't see the Auth table in the table list. Am I missing something here? I prefer doing the user table update on the first signup request on server side via a trigger attached to Signup request.
<img width="653" alt="Screen Shot 2021-08-18 at 10 55 03 PM" src="https://user-images.githubusercontent.com/4299288/130005441-0c6eab6d-7fd5-40ca-9746-c287fbd838cf.png">

**Comment by inian** [comment, marked as answer]
Track the issue here @CR1AT0RS  - https://github.com/supabase/supabase/issues/2690

**Reply by CR1AT0RS** [reply]
Can I also send raw data with auth.signup(email, password, raw data)? 
raw_user_meta_data  --> It looks like this comes with Oauth requests. Can I send this raw data with regular signup() function?

begin
  insert into public.users (id, full_name, avatar_url)
  values (new.id, new.raw_user_meta_data->>'full_name', new.raw_user_meta_data->>'avatar_url');
